python/src/KrigingAlgorithm_doc.i.in

```diff
@@ -157,7 +157,7 @@ Create the Kriging algorithm with the optimizer option:
 >>> bounds = ot.Interval(1e-2,1e2)
 >>> optimizer.setBoundConstraints(bounds)
```

Comment:
The OptimizationSolver class does not provide such service
You should first define an optimization problem, fix bounds and set the problem to the solver